plugins/aggregators/derivative/sample.conf

```diff
@@ -1,17 +1,41 @@
 # Calculates a derivative for every field.
 [[aggregators.derivative]]
-  ## Specific Derivative Aggregator Arguments:
-
-  ## Configure a custom derivation variable. Timestamp is used if none is given.
-  # variable = ""
-
-  ## Suffix to add to the field name for the derivative name.
+  ## The period in which to flush the aggregator.
+  period = "30s"
+  ##
+  ## If true, the original metric will be dropped by the
```

Comment:
Did we move this detailed description out of the sample config on purpose because it takes a lot of space? I think the less detailed one is better

Reply:
I don't mean to defeat the purpose of embedding. The sample.conf should have a quick description of each setting. The readme.md is the right place for most of the details of what the plugin does and how to use it. If the details in sample.conf aren't in readme.md, could you move them there?

If we put details in comments in each plugin's toml, then when we merge them into the full etc/telegraf/telegraf.conf in the release it would be too big.

We can leave out `drop_original` since it applies to all aggregators and it's documented in https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#aggregator-plugins